File: modules/opla-front/src/app/mlconfig/mlmodels/LMSLearningModel.ts
import {MachineLearningModel} from "./MachineLearningModel";
import {MachineLearningAlgorithm} from "./MachineLearningAlgorithm";

export class LMSLearningModel extends MachineLearningModel{

  sampleSize: string;

  constructor(sampleSize: string) {
    super(MachineLearningAlgorithm.LMS);
    this.sampleSize = sampleSize;
  }
}

File: modules/opla-core/src/main/java/br/otimizes/oplatool/core/learning/mlmodels/LMSLearningModel.java
package br.otimizes.oplatool.core.learning.mlmodels;

public class LMSLearningModel extends MachineLearningModel {
    private int sampleSize;

    public LMSLearningModel() {
    }

    public LMSLearningModel(int sampleSize) {
        this.sampleSize = sampleSize;
    }

    public int getSampleSize() {
        return sampleSize;
    }

    public void setSampleSize(int sampleSize) {
        this.sampleSize = sampleSize;
    }
}
